analysis_plan.md

```diff
@@ -0,0 +1,27 @@
+
+- The parse_fasta and parse_header functions, in order to traverse the data from the file
+- The kmer composition one from assignment 4
+- A function that can compare the sequences from each kmer across groups of data, keeping track of how many kmers early Africa and late Africa data have in common and also how many kmers early Africa and South America have in common
+
+I could present this information using a line plot that records the size of the kmer against the amount of kmers in common for early Africa against late Africa and early Africa against South America. The lines may vary as the kmer sizes do, but lines that are higher up on the graph will obviously indicate more kmers in common and therefore may be indicators that these two groups of data are more similar than the other.
+
+# Plan 2
+I am interested in developing a phylogenetic tree of the COVID virus located in Asia from December 2019 to the most current data. I am limiting by these factors in order to try to keep the amount of data that I am analyzing to smaller, easier to work with amount. Based off of research largely from biointeractive.org (https://www.biointeractive.org/classroom-resources/creating-phylogenetic-trees-dna-sequences) and the Oxford University Press youtube (https://www.youtube.com/watch?v=09eD4A_HxVQ), the basic steps to making a tree seem to be:
+
+
+- Start by finding the sequences with the least amount of differences between them, these can be branched together as most likely to be closely linked
+- Then essentially work backwards to compare individual sequences to the ones already graphed, picking the ones with the least differences to add to the graph as you go
+- For the entire process, keep in mind that if difference scores are equal at any point, will need to find a way to average them
+
+
+ Functions that I will need to make use of are:
+- A scoring algorithm; I believe I can use Needleman-Wunsch for this by taking the score from the bottom right of the scored matrix as the overall score for the difference between the sequences. A higher number would reflect more similar sequences

```

Comment:
You could, might take a long time to calculate, I'm not sure. Might also be worth trying `SmithWaterman` instead, since it's not 100% clear that all of the genomes are in the same orientation, or at the same level of completeness. 

Alternatively, you could just use your kmer functions from above for a quick-and-dirty distance metric